doc/radosgw/vault.rst

```diff
@@ -255,12 +393,44 @@ restricted to fetch KV keys as follows::
 
   rgw crypt vault prefix = /v1/secret/data
 
-Or, in the case of exportable transit keys::
+Or, when using the transit secret engine::
 
-  rgw crypt vault prefix = /v1/transit/export/encryption-key
+  rgw crypt vault prefix = /v1/transit
 
 In the example above, the Gateway would only fetch transit encryption keys under
-``http://vault-server:8200/v1/transit/export/encryption-key``.
+``https://vault-server:8200/v1/transit``.
+
+Transit engine compatibility support
+------------------------------------
+The transit engine has compatibility support for previous
+versions of ceph, which used the transit engine as a simple key store.
+
+There is a a "compat" option which can be given to the transit
+engine to configure the compatibility support,
```

Comment:
can you please add a note about this to `PendingReleaseNotes`? we want existing users to know about the new/recommended configuration